core/src/main/java/org/dst/core/operatorImpl/DstListImpl.java

```diff
@@ -1,21 +1,27 @@
 package org.dst.core.operatorImpl;
 
-import org.dst.core.exception.NotImplementException;
 import org.dst.core.operatorset.DstList;
+import org.dst.utils.enums.Status;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 import java.util.HashMap;
 import java.util.List;
 
 public class DstListImpl implements DstList {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(DstListImpl.class);
+
   private HashMap<String, List<String>> listMap;
 
   public DstListImpl() {
     this.listMap = new HashMap<String, List<String>>();
   }
 
   @Override
-  public void put(String key, List<String> value) {
+  public Status put(String key, List<String> value) {
     listMap.put(key, value);
+    return Status.OK;
   }
 
   @Override
```

Comment:
We should judge whether the `key` is null or not.

Reply:
okay